include/oboe/AudioStreamBuilder.h

```diff
@@ -499,6 +499,16 @@ class AudioStreamBuilder : public AudioStreamBase {
                 || (mAudioApi == AudioApi::Unspecified && isAAudioRecommended());
     }
 
+    /**
+     * @return true if MMAP will be used based on the current settings.
```

Comment:
We don't know if MMAP will be used until we call AAudio. We can only report whether it is supported on the device.